website/docs/guides/delivery.mdx

```diff
@@ -8,27 +8,60 @@ import TierLabel from "../_components/TierLabel";
+This guide uses Flux manifests to install Flagger and Linkerd. Flagger can work with a number of
+service meshes and ingress controllers, to support various progressive delivery [deployment
+strategies](https://docs.flagger.app/usage/deployment-strategies):
+
+![canary release icon](/img/canary.svg)  **Canary Release** - where traffic is gradually shifted to
+the new version and its performance is assessed. Based on this analysis of KPIs a release is either
+promoted or the update abandonned.  
+![a b testing icon](/img/ab.svg)  **A/B Testing** - uses HTTP headers or cookies to ensure users
+  stay on the same version of an application during the canary analysis.  
+![blue green testing icon](/img/blue-green.svg)  **Blue/Green** - where tests are run against the
+  new version, and if successful, traffic is then switched from the current service.  
+![blue green mirroring icon](/img/mirroring.svg)  **Blue/Green with Traffic Mirroring** - sends
+  copies of incoming requests to the new version. The user recieves the response from the current
+  service and the other is discarded. The new version is promoted only if metrics are healthy.
+
+Using Flux allows us to manage our cluster applications in a declarative way through changes in a
+Git repository.
 
 In this guide, we will walk you through a full end-to-end scenario where you will:
 - [Install the Linkerd service mesh](#installing-linkerd-using-flux)
 - [Install Flagger](#installing-flagger-using-flux)
-- [Deploy a sample application using a canary release strategy based on metrics provided through Linkerd's in-built Prometheus instance](#deploy-a-canary-release)
+- [Deploy a sample application using a canary release strategy based on metrics provided through
+  Linkerd's in-built Prometheus instance](#deploy-a-canary-release)
 
 ## Prerequisites
-- This guide assumes you already have a Kubernetes cluster running and have bootstrapped Flux. To apply the manifests listed in this guide, you will need to commit them to a repository being reconciled with Flux. For help installing Flux, you can follow their [getting started](https://fluxcd.io/docs/get-started/) documentation.
-- Flagger requires the `autoscaling/v2` API to be installed on the cluster, you can use `kubectl api-resources` to check which API versions are supported.
-- The [step](https://smallstep.com/cli/) CLI installed to generate certificates in order to support mTLS connections.
+- This guide assumes you already have a Kubernetes cluster running and have bootstrapped Flux. To
+  apply the manifests listed in this guide, you will need to commit them to a repository being
+  reconciled with Flux. For help installing Flux, you can follow their [getting
+  started](https://fluxcd.io/docs/get-started/) documentation.
+- Flagger requires the `autoscaling/v2` API to be installed on the cluster, you can use `kubectl
```

Comment:
I believe Flagger also supports `autoscaling/v2beta2` API so maybe we should correct this here.